doc/release/0.18.0-notes.rst

```diff
@@ -62,6 +62,10 @@ is now consistently added after the matrix is applied,
 independent of if the matrix is specified using a one-dimensional
 or a two-dimensional array.
 
+The `mode` keyword argument in ``signal.spectrogram`` no longer supports the
```

Comment:
These should probably be single-backticks instead of double -- Sphinx should be able to find these functions. Feel free to fix the ones above, too.